resources/todomvc/big-dom-generator/utils/buildComplex.js

```diff
@@ -7,7 +7,7 @@ const COMPLEX_DOM_HTML_FILE = "index.html";
 const TODO_HTML_FILE = "index.html";
 const CSS_FILES_TO_ADD_LINKS_FOR = ["big-dom-generator.css", "generated.css"];
 
-function buildComplex(CALLER_DIRECTORY, SOURCE_DIRECTORY, TITLE, FILES_TO_MOVE) {
+function buildComplex(CALLER_DIRECTORY, SOURCE_DIRECTORY, TITLE, FILES_TO_MOVE, EXTRA_CSS_TO_LINK = [], SCRIPTS_TO_LINK = []) {
```

Comment:
This will need to be updated with the new way of passing parameters in the other patch.